code/post.r

```diff
@@ -75,9 +72,9 @@ posterior.sim <- function(data, model, over = FALSE) {
           }
 
         }
-      }
-      oo <- c(rep(0, data$str[ii] - 1), oo)
+      } 
     }
+    oo <- c(rep(0, data$str[ii] - 1), oo)
```

Comment:
moved 0 pad outside time (jj) loop. before was padding each jj with many zeros, which ballooned the series out of control. I think this is correct across all time series.